### ISSUE TITLE
[Question] LwM2M no request from server after registration

### ISSUE THREAD

**Issue by etiennedm** [thingsboard/thingsboard#6179]
**Component**
 * LwM2M server

**Description**
After following [the instructions to setup a LwM2M device](https://thingsboard.io/docs/reference/lwm2m-api/), I try to connect a Wakaama client by running `./lwm2mclient -n wakaama-client -p 5685`. I can see the registration happening, both from the Wakaama client output, and the Thingsboard logs. I made sure to enable the Device/Manufacturer, Device/Model number, Device/Serial number and Device/Battery level attributes, as well as the Device/Battery level telemetry and observation. But nothing happens after registration, no request for the resources I selected in the profile configuration. 

I can also run the `quit` command in the Wakaama client, and the device is properly unregistered. I can see the packets in Wireshark and nothing looks suspicious, except that there is no traffic going on from the server to fetch the device data.

What am I missing? Thanks for your help!

**Environment**
 * OS: Docker image on Ubuntu 18.04.6
 * ThingsBoard: 3.3.3

**Comment by etiennedm** [comment]
I moved on to the RPC section to see what would happen, and here are my observations:

- `Read {"id":"/3/0/9"}` works returning `{"result":"CONTENT","value":"LwM2mSingleResource [id=9, value=72.0, type=FLOAT]"}` as expected.
- `Read {"key":"batteryLevel"}` returns `{"result":"BAD_REQUEST","error":"batteryLevel is not configured in the device profile!"}`

Yet the `batteryLevel` key does exist in the device profile and is assigned to the /3/../9 resource id aka. battery level. I hope this can help us troubleshoot the issue.

**Comment by etiennedm** [comment]
I figured it out, I was using version 1.2 of the object that I got here [in the LwM2M registry on Github](https://github.com/OpenMobileAlliance/lwm2m-registry). If I use instead the version 1.0 found on [this page](https://technical.openmobilealliance.org/OMNA/LwM2M/LwM2MRegistry.html), everything seems to work as expected.

Is that normal? What can we do to help the used troubleshoot this issue more easily (I've been scratching my head about this for the last few days)?
